### ISSUE TITLE
Installation failed on Laravel v8.12

### ISSUE THREAD

**Issue by christophstockinger** [knuckleswtf/scribe#131]
- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.readthedocs.io/en/latest/troubleshooting.html).**

**What happened?**
I wanted to install Scribe using composer, but the following incompatibilities occurred, making it impossible to use Scribe.

```
Problem 1
    - Installation request for knuckleswtf/scribe 2.0.0 -> satisfiable by knuckleswtf/scribe[2.0.0].
    - Conclusion: remove fakerphp/faker v1.10.1
    - Conclusion: don't install fakerphp/faker v1.10.1
    - knuckleswtf/scribe 2.0.0 requires fzaninotto/faker ^1.9.1 -> satisfiable by fakerphp/faker[1.9.x-dev, v1.9.1], fzaninotto/faker[1.9.x-dev, v1.9.1].
    - Can only install one of: fakerphp/faker[1.9.x-dev, v1.10.1].
    - Can only install one of: fakerphp/faker[v1.9.1, v1.10.1].
    - fzaninotto/faker 1.9.x-dev conflicts with fakerphp/faker[v1.10.1].
    - fzaninotto/faker v1.9.1 conflicts with fakerphp/faker[v1.10.1].
    - Installation request for fakerphp/faker (locked at v1.10.1, required as ^1.9.1) -> satisfiable by fakerphp/faker[v1.10.1].
```

**My environment:**
 - PHP version (from `php -v`):  7.3.21
 - Framework (Laravel): 8.12
 - Laravel/Lumen version (from `composer show laravel/framework`):  v8.12.3
 - Scribe version (from `composer show knuckleswtf/scribe`): 2.0.0

**Additional info:**
My installed Composer Packages
````
...
"require": {
        "php": "^7.3|^8.0",
        "fideloper/proxy": "^4.4",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^8.12",
        "laravel/sanctum": "^2.7",
        "laravel/tinker": "^2.5"
    },
    "require-dev": {
        "facade/ignition": "^2.5",
        "fakerphp/faker": "^1.9.1",
        "mockery/mockery": "^1.4.2",
        "nunomaduro/collision": "^5.0",
        "phpunit/phpunit": "^9.3.3"
    },
...
```

**Comment by shalvah** [comment]
I guess it's because Laravel has ported from fzaninotto/faker to fakerphp/faker (the successor). Changing the name in our composer.json should fix it.

Can't do this atm, but you can send in a PR.

**Comment by shalvah** [comment]
This should be fixed in 2.0.3
